packages/@lwc/engine/scripts/rollup/rollup.config.js

```diff
@@ -45,6 +48,7 @@ function rollupConfig({ format = 'es' } = {}) {
                 typescript,
                 include: TS_WHITELIST,
             }),
+            rollupLwcFeatures({ featureFlags }),
```

Comment:
Why do we need to have 2 packages the distributed version of the engine don't have any reference to the original feature flags in the code?

Reply:
I didn't understand this feedback, could you rephrase?

Reply:
Sorry.

I was wandering, who else except the engine would consume the feature flag package?

Reply:
The compiler could eventually consume the package as well.